Update version.md (#6372)
fishtown-analytics/docs.getdbt.com: website/docs/reference/commands/version.md

## Versioning
To learn more about release versioning for dbt Core, refer to [How dbt Core uses semantic versioning](/docs/dbt-versions/core#how-dbt-core-uses-semantic-versioning). 

> If using [versionless dbt Cloud](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless), then `dbt_version` uses the latest (continuous) release version. This also follows semantic versioning guidelines, using the `YYYY.MM.DD+<suffix>` format, where the year, month, and day represent the date the version was built (e.g., `2024.10.28+996c6a8`). The suffix provides additional unique identification for each build.

Reviewer: ```suggestion
If using [versionless dbt Cloud](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless), then `dbt_version` uses the latest (continuous) release version. This also follows semantic versioning guidelines, using the `YYYY.MM.DD+<suffix>` format. The year, month, and day represent the date the version was built (for example, `2024.10.28+996c6a8`). The suffix provides an additional unique identification for each build.
```

> If using [versionless dbt Cloud](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless), then `dbt_version` uses the latest (continuous) release version. This also follows semantic versioning guidelines, using the `YYYY.MM.DD+<suffix>` format, where the year, month, and day represent the date the version was built (e.g., `2024.10.28+996c6a8`). The suffix provides additional unique identification for each build."

Author: ```suggestion
If using [versionless dbt Cloud](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless), then `dbt_version` uses the latest (continuous) release version. This also follows semantic versioning guidelines, using the `YYYY.MM.DD+<suffix>` format, where the year, month, and day represent the date the version was built (e.g., `2024.10.28+996c6a8`). The suffix provides additional unique identification for each build.
```
